### ISSUE TITLE
Create Action errors have different error types

### ISSUE THREAD

**Issue by Bagera** [zetkin/organize.zetk.in#760]
If you leave out Campaign you get a 404, this should be 400 as with other errors.

**Comment by richardolsson** [comment]
This is actually a bug in the Zetkin Platform. Will be fixed in R14.

**Comment by richardolsson** [comment]
Turns out it's not a bug in the Platform. The API requires actions to be posted to /campaigns/{campaign_id}/actions, and if the campaign ID does not exist (or is undefined, as in this case) it will return a 404, as is to be expected.

This will need to be solved by checking for undefined before making the request. If we want to get the same message we get when we submit an action where activity or location is undefined, we need to implement that logic in the error message store. But this is a bit hackish and perhaps a waste of time.

Or, we could just leave this as is (albeit not ideal) until we've implemented proper validation as suggested by #43.

To be decided during this sprint.
